woocommerce/woocommerce-android
# Fix Print Shipping Label Navigation Issue

Closes: #10924


### Description

When you go through the flow of purchasing a shipping label and you print it and you are pressing back to go to the order details you would be taken to the home screen i.e My Store. This PR fixes this issue so you are taken to order details.

### Testing instructions

Verify the below on tablets and phones. I have noticed inconsistencies with this navigation when used on either device before this fix. 

For these tests it is recommended to do the flows below on `trunk` for both phones and tablets to see the behavior of the app before switching to this branch. 

#### Printing 

1. Go to an Order that is eligible for shipping labels.
2. Go through the flow to the purchased label being printed.
3. Attempt to press back to go back to order details.
4. Notice you end up on order details. 

#### Reprint 

1. Go to an Order that has a shipping label that has been printed already.
2. Go to the print the order.
3. Press back to go back to order details.
4. Notice you end up there. 

### Images/gif

## The bug that is taking place on Tablets

[printing-bug.webm](https://github.com/woocommerce/woocommerce-android/assets/1509205/60bebbd2-c6a1-4ec5-a30c-40f64a01420f)

## Printing Flow 

[printing.webm](https://github.com/woocommerce/woocommerce-android/assets/1509205/6897b34d-ade0-488a-b993-8c9355fcb51f)

## Reprinting Flow

[reprint.webm](https://github.com/woocommerce/woocommerce-android/assets/1509205/3e585373-e73b-4fb4-a156-0015f03afdf4)
